components/cli/internal/test/kubecli.go

```diff
@@ -27,24 +27,32 @@ import (
 type MockKubeCli struct {
 	cells      kubernetes.Cells
 	composites kubernetes.Composites
+	cellsBytes map[string][]byte
 }
 
-type option func(*MockKubeCli)
+func (kubeCli *MockKubeCli) SetVerboseMode(enable bool) {
+}
 
-func WithCells(cells kubernetes.Cells) option {
+func WithCells(cells kubernetes.Cells) func(*MockKubeCli) {
 	return func(cli *MockKubeCli) {
 		cli.cells = cells
 	}
 }
 
-func WithComposites(composites kubernetes.Composites) option {
+func WithComposites(composites kubernetes.Composites) func(*MockKubeCli) {
 	return func(cli *MockKubeCli) {
 		cli.composites = composites
 	}
 }
 
+func WithCellsBytes(cellsBytes map[string][]byte) func(*MockKubeCli) {
```

Comment:
suggestion: rename the method to WithCellsAsBytes.

Reply:
Updated to WithCellsAsBytes.